### PR TITLE
add saiserver docker for mlnx sn2700 platform

### PR DESCRIPTION
**Pull request by lguohan**

### CODE REVIEW

File: dockers/docker-saiserver-mlnx/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+FROM docker-base
+
+RUN apt-get update
+
+COPY deps /deps
```
Comment (qiluo-msft) [inline dockers/docker-saiserver-mlnx/Dockerfile]: More specific files in deps/

File: dockers/docker-saiserver-mlnx/portmap.ini [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Ethernet23      88,89,90,91
+Ethernet24      92,93,94,95
+Ethernet25      96,97,98,99
+Ethernet26      100,101,102,103
+Ethernet27      104,105,106,107
+Ethernet28      108,109,110,111
+Ethernet29      112,113,114,115
+Ethernet30      116,117,118,119
+Ethernet31      120,121,122,123
+Ethernet32      124,125,126,127
```
Comment (qiluo-msft) [inline dockers/docker-saiserver-mlnx/portmap.ini]: Add new line at the end?

File: dockers/docker-saiserver-mlnx/profile.ini [hunk 3]
```diff
@@ -0,0 +1 @@
+SAI_INIT_CONFIG_FILE=/usr/share/sai_2700.xml
```
Comment (qiluo-msft) [inline dockers/docker-saiserver-mlnx/profile.ini]: Add new line at the end?

File: dockers/docker-saiserver-mlnx/sai_2700.xml [hunk 4]
```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0"?>
+<root>
+	<platform_info type="2700">
+
+		<!-- Device MAC address  -->
```
Comment (qiluo-msft) [inline dockers/docker-saiserver-mlnx/sai_2700.xml]: Replace tab with spaces?

File: dockers/docker-saiserver-mlnx/start.sh [hunk 5]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+function clean_up {
+    service rsyslog stop
+    exit
```
Comment (qiluo-msft) [inline dockers/docker-saiserver-mlnx/start.sh]: No need to 'exit'

File: dockers/docker-saiserver-mlnx/start.sh [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+start_mlnx()
+{
+    mkdir -p /dev/sxdevs
```
Comment (qiluo-msft) [inline dockers/docker-saiserver-mlnx/start.sh]: Move after '-e' test? Or you may create the folder in Dockerfile.

File: dockers/docker-saiserver-mlnx/Dockerfile (hunk 1 above)
Comment (lguohan) [inline dockers/docker-saiserver-mlnx/Dockerfile]: fixed

File: dockers/docker-saiserver-mlnx/portmap.ini (hunk 2 above)
Comment (lguohan) [inline dockers/docker-saiserver-mlnx/portmap.ini]: I prefer not to have newline in the end.

File: dockers/docker-saiserver-mlnx/start.sh (hunk 6 above)
Comment (lguohan) [inline dockers/docker-saiserver-mlnx/start.sh]: that's what syncd does.

File: dockers/docker-saiserver-mlnx/sai_2700.xml (hunk 4 above)
Comment (lguohan) [inline dockers/docker-saiserver-mlnx/sai_2700.xml]: this file is from mlnx, do not want to change it.

File: dockers/docker-saiserver-mlnx/start.sh (hunk 6 above)
Comment (lguohan) [inline dockers/docker-saiserver-mlnx/start.sh]: moved

**Pull request merged**